src/beast_feast.py

```diff
@@ -0,0 +1,6 @@
+def beast_feast(beast, dish):
+    return beast[0].casefold() == dish[0].casefold()and dish[-1].casefold() == beast[-1].casefold()
```

Comment:
Needs a space between `)and`. Run `black` on your code before committing and opening the pull requests.

Reply:
okay thank you